core/frontend/src/tools/AccuDrawViewportUI.ts

```diff
@@ -46,39 +46,97 @@ export class AccuDrawViewportUI extends AccuDraw {
     simplifiedInput: true,
     /** Enable simple math operations not supported by quantity parser. */
     mathOperations: true,
-    /** Number of visible characters to show in text input fields. */
-    fieldSize: 12,
-    /** Row spacing of text input fields for vertical arrangement. */
-    rowSpacingFactor: 1.2,
-    /** Column spacing of text input fields and buttons for horizontal arrangement. */
-    columnSpacingFactor: 1.1,
-    /** Corner radius of text input fields and locks buttons. */
-    borderRadius: "0.5em",
-    /** Background color of unfocused text input fields and unlocked buttons. */
-    backgroundColor: "rgba(150, 150, 150, 0.5)",
-
-    /** Settings specific to text input fields and lock button labels. */
-    text: {
-      /** Font family to use for text input field values and button labels. */
-      fontFamily: "sans-serif",
-      /** Font size to use for text input field values and button labels. */
-      fontSize: "9pt",
-      /** Font color to use for text input field values and button labels. */
-      color: "white",
-      /** Background color of focused text input field. */
-      focusColor: "rgba(50, 50, 200, 0.75)",
+    /** Settings that apply to both text input fields and lock buttons. */
+    field: {
+      /** Number of visible characters to show in text input fields. */
+      size: 12,
+      /** Height of text input fields and lock buttons. */
+      height: "var(--iui-component-height-small, 1.75em)",
```

Comment:
what does all this look like if someone isnt using itwinui?
@GerardasB can you test that out?
instead of using hardcoded itwinui vars, could we use the actual resolved values? is that possible

Reply:
@aruniverse @GerardasB I tested it with display-test-app, it looks the same because the fallbacks use the css variable value...